components/layout_2020/query.rs

```diff
@@ -531,6 +514,29 @@ fn process_offset_parent_query_inner(
     })
 }
 
+/// Returns whether or not the element with the given style and body element determination
+/// is eligible to be a parent element for offset* queries.
+///
+/// From <https://www.w3.org/TR/cssom-view-1/#extensions-to-the-htmlelement-interface>:
```

Comment:
... and then https://www.w3.org/TR/cssom-view-1/#dom-htmlelement-offsetparent would be a more specific link.

Reply:
Yeah, even if this applies to all `offset` queries, this link is better and I've updated it.